### PR TITLE
Chapter 1 completed

### PR DESCRIPTION
**Pull request by lprakashv**
### Solutions for Chapter One

cc @vrom911 @chshersh

### CODE REVIEW

File: src/Chapter1.hs [hunk 1]
```diff
@@ -556,7 +562,14 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
```
[the remark this answers is not in the document]
Comment (lprakashv) [reply src/Chapter1.hs]: Fixed

File: src/Chapter1.hs [hunk 2]
```diff
@@ -508,7 +508,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if (abs x) < (abs y) then x else y
```
[the remark this answers is not in the document]
Comment (lprakashv) [reply src/Chapter1.hs]: Fixed

[the remark this answers is not in the document]
Comment (lprakashv) [reply src/Chapter1.hs]: Fixed

File: src/Chapter1.hs [hunk 3]
```diff
@@ -619,8 +631,13 @@ Implement a function that returns the sum of the last two digits of a number.
...
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastDig (abs n) + lastButOneDig (abs n)
+  where
+    lastDig :: Int -> Int
+    lastDig x = mod x 10
+    lastButOneDig :: Int -> Int
+    lastButOneDig x = mod (div x 10) 10
```
Comment (chshersh) [inline src/Chapter1.hs:640]: That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂

**Review by chshersh — APPROVED** [review]
Great solutions for Chapter One! And well done with the Advanced Task :clap: 

The solutions look good :+1: I'm going to close this PR, because we can't merge the solutions to our repo :slightly_smiling_face: But don't worry, it still counts towards Hacktoberfest contribution :shirt: :deciduous_tree: Feel free to open PRs with solutions to other chapters!

File: src/Chapter1.hs (hunk 3 above)
Comment (lprakashv) [reply src/Chapter1.hs:640]: Hey, thanks a lot. I just wanted nudge to learn Haskell, this repo proved to be a really great exercise! Will push the other PR's soon after the weekend. :)

**Pull request closed without merging**